packages/web/src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -428,17 +429,20 @@ const TrackHeader = ({
         />
       ) : null}
       {streamConditions && trackId ? (
-        <GatedTrackSection
-          isLoading={isLoading}
-          trackId={trackId}
-          streamConditions={streamConditions}
-          hasStreamAccess={hasStreamAccess}
-          isOwner={isOwner}
-          wrapperClassName={styles.gatedTrackSectionWrapper}
-          className={styles.gatedTrackSection}
-          buttonClassName={styles.gatedTrackSectionButton}
-          ownerId={userId}
-        />
+        <Box mb='xl'>
```

Comment:
Just curious why removing the margins was necessary for `GatedContentSection`?

Reply:
The margin was specific to its existing usages, but resulted in too much margin in the collection page.
Better to just apply the margin on the usage instead of the component I think.